src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -175,19 +175,40 @@
     private static final Pattern JAVADOC_MULTILINE_TO_SINGLELINE_PATTERN =
             Pattern.compile("\n[ ]+(\\*)|^[ ]+(\\*)");
 
-    /** Period literal. */
+    /**
```

Comment:
please revert all changes that are not required for functional fix.

Reply:
done